tests/test_SklearnSVMConverters.py

```diff
@@ -78,13 +118,70 @@ def test_convert_svmc_linear_binary(self):
             X,
             model,
             model_onnx,
-            basename="SklearnBinSVCLinearPF",
-            allow_failure=True,
+            basename="SklearnBinSVCLinearPF-NoProbOpp",
+            allow_failure="StrictVersion(onnxruntime.__version__)"
+                          " <= StrictVersion('0.5.0')"
```

Comment:
Will the changes be made in ONNX Runtime by the version 0.5.0 release? Perhaps < is more appropriate that <=

Reply:
Good call :), the PR is already in onnxruntime so < is better.